chia/consensus/blockchain.py

```diff
@@ -76,7 +76,10 @@ class StateChangeSummary:
     peak: BlockRecord
     fork_height: uint32
     rolled_back_records: List[CoinRecord]
-    new_npc_results: List[NPCResult]
+    # list of coin-id, puzzle-hash pairs
+    removals: List[Tuple[bytes32, bytes32]]
+    # new coin and hint
+    additions: List[Tuple[Coin, Optional[bytes]]]
```

Comment:
A hint should be bytes32 yeah?

Reply:
actually, this is populated with any length bytes as long as it doesn't exceed 32 bytes